src/views/search_home.hbs

```diff
@@ -109,14 +109,14 @@
           </select>
 
           <label for="signup-reason" class="f5 b tl db mb2">Reason for signing up* </label>
-          <select id="signup-reason" name="signup-reason" class="mb2 pa2 w-100 bg-light-gray br2" required="">
+          <select id="signup-reason" name="signup-reason" class="mb2 pa2 w-100 bg-light-gray br2"  required="">
             <option value=""></option>
             <option value="Investment">Investment</option>
             <option value="Hire-someone">Hire someone</option>
             <option value="Find-client">Find clients</option>
             <option value="Other">Other</option>
           </select>
-
+          <input type="text" name="other" id="other" class="input-reset bg-light-gray ba b--black-20 pa2 mb2 w-100 br2 dn" value="">
```

Comment:
what if the dropdown list is opened and the user clicks other? would the user be able to see the input ? or would it be hidden by the dropdown list ?

Reply:
run it yourself and find out mate :sunglasses:

Reply:
I did and found a different bug.
that when you press any option ( not just other ) it shows the input.
for example if you choose "Find clients" the input shows up